lib/models/tree-node.model.ts

```diff
@@ -392,10 +433,10 @@ export class TreeNode implements ITreeNode {
     this.treeModel.setFocus(true);
 
     const actionMapping = this.options.actionMapping.mouse;
-    const action = actionMapping[actionName];
+    const mouseAction = actionMapping[actionName];
 
-    if (action) {
-      action(this.treeModel, this, $event, data);
+    if (mouseAction) {
+      action(this.treeModel, $event, data);
```

Comment:
You renamed the variable but didn't change the references to it.
You should rename variables using the IDE (like in VSCode you have "rename symbol" option).
This also means that you didn't really run the tests, because it's impossible they passed with this code

Reply:
Fixed; Same as above

Reply:
Cool. Can you push?